glossary.ftd

```diff
@@ -1,22 +1,219 @@
 -- ft.page: Glossary
 
-For readers/writers of document by id, we have three kinds of id: `id: /foo/`,
-`full-id: /foo/-/x/`, `filename: foo.ftd`.
+An FPM Glossary
 
 
-Since we need to get `id`, `full-id` and `filename` of current document, to power
-document-readers, lets create processors `document-id`, `document-full-id` and
-`document-filename` as helpers as well. These three processors will return ids
-for current document.
+
+-- ft.h2: client project
+
+An [fpm project](glossary/#fpm-project) which is a clone to the
+[server project](glossary/#server-project) and tracks it. The interaction
+with server project is established using [sync](glossary/#sync).
+
+
+-- ft.h2: clone
+
+Clone a [server project](glossary/#server-project) into a new directory
+which becomes a [client](glossary/#server-project).
+
+
+-- ft.h2: conflict-state
```

Comment:
Not a good name. Term should be things you can use in a grammar without having to add other terms to make its meaning clear. Here you have to `a file is in "conflict state"`, this entire phrase only makes sense, `conflict-state` does not. I prefer calling it `conflicted-file`.

Reply:
Or `file-with-conflict`.

Reply:
`conflicting-file`. I think `file-with-conflict` is my preference.